pyproject.toml

```diff
@@ -122,7 +122,6 @@ filterwarnings = [
   "error",
   "ignore:path is deprecated.:DeprecationWarning",
   "ignore:The --rsyncdir command line argument and rsyncdirs config variable are deprecated.:DeprecationWarning",
-  "ignore:Python 3.14 will, by default, filter extracted tar archives:DeprecationWarning",
```

Comment:
I think flit is triggering this? Pretty sure it's not us. I might try only ignoring it for Flit, if that doesn't work, I'll just restore it.

Reply:
Ahh, it's actually in our tests. Okay, will put it back for now. We maybe can silence this in our tests in the future.